package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "thrift2flow",
-  "version": "0.2.12",
+  "version": "0.2.13",
```

Comment:
don't bump package version, that will happen after the pull request is landed and a new version is released.